README.md

```diff
@@ -156,6 +168,8 @@ You can skip the comment sending by omitting the option or by passing an empty s
 
 Default value: unset
 
+Required Permission: `issues: write`
```

Comment:
```suggestion
Default value: unset
Required Permission: `issues: write`
```
I think removing this extra row would make it more readable.
It's a minor change, but I would rather have something a little more compact! WDYT?

Reply:
But that makes it all on one line?!

Reply:
Not if you use the double space at the end of the row